frontend/src/metabase/admin/routes.jsx

```diff
@@ -182,6 +182,7 @@ const getRoutes = (store, CanAccessSettings, IsAdmin) => (
           ))}
         </Route>
       </Route>
+      {PLUGIN_METABOT.AdminRoute}
```

Comment:
very proud of this new method of plugging in an enterprise route, it makes it clear what might be there rather than some mystery array